### DISCUSSION TITLE
SAP Privileges version roll-back

### DISCUSSION THREAD

**Discussion by LukeMutlow21** [SAP/macOS-enterprise-privileges#136] [General]
Hi,

I manage a Jamf Pro MDM, in which we use the Mac Apps section for application patching. Is there any reason why the SAP Privileges app has gone from version 2.0.0 back to 1.5.4?

Kind regards,

Luke Mutlow

**Comment by mthielemann** [comment]
@LukeMutlow21  Wouldn't this be a question to ask Jamf? As I know, Jamf pulled version 2 because they had issues with their App Catalog but there should be an updated version be available soon.

**Reply by LukeMutlow21** [reply]
Good to know, thank you @mthielemann
